app/models/checkout.rb

```diff
@@ -4,6 +4,8 @@ class Checkout < ApplicationRecord
   validates :user, :media_object_id, :checkout_time, :return_time, presence: true
 
   scope :active_for, ->(media_object_id) { where(media_object_id: media_object_id).where("return_time > now()") }
+  scope :checkouts, ->(user_id) { where(user_id: user_id).where("return_time > now()") }
+  scope :returned_checkouts, ->(user_id) { where(user_id: user_id).where("return_time < now()") }
```

Comment:
What do you think about making these scope names have the param scope in them?
```suggestion
  scope :active_for_media_object, ->(media_object_id) { where(media_object_id: media_object_id).where("return_time > now()") }
  scope :active_for_user, ->(user_id) { where(user_id: user_id).where("return_time > now()") }
  scope :returned_for_user, ->(user_id) { where(user_id: user_id).where("return_time < now()") }
```